simpa/core/simulation_modules/__init__.py

```diff
@@ -1,3 +1,44 @@
 # SPDX-FileCopyrightText: 2021 Division of Intelligent Medical Systems, DKFZ
 # SPDX-FileCopyrightText: 2021 Janek Groehl
 # SPDX-License-Identifier: MIT
+
+from abc import abstractmethod
+
+from simpa.core import PipelineModule
+from simpa.utils import Settings
+
+class SimulationModule(PipelineModule):
+    """
+    Defines a simulation module that is a step in the simulation pipeline.
```

Comment:
Please add that a `SimulationModule` is one of these 4: Volume Creation, Light Propagation Modeling, Acoustic Wave Propagation Modeling, Image Reconstruction